source/agora/consensus/data/Block.d

```diff
@@ -449,15 +450,16 @@ public Block makeNewBlock (Transactions)(const ref Block prev_block,
     block.txs.sort;
 
     block.header.merkle_root = block.buildMerkleTree();
+    block.header.enrollments = enrollments;
 
     return block;
 }
 
 ///
 @safe nothrow unittest
 {
-    auto new_block = makeNewBlock(GenesisBlock, [Transaction.init]);
-    auto rng_block = makeNewBlock(GenesisBlock, [Transaction.init].take(1));
+    auto new_block = makeNewBlock(GenesisBlock, [Transaction.init], [Enrollment.init]);
+    auto rng_block = makeNewBlock(GenesisBlock, [Transaction.init].take(1), [Enrollment.init]);
```

Comment:
Adding an invalid Enrollment doesn't seem like the best test

Reply:
Modify to correct value.

Reply:
The part I was worried about is whether I should align here once more.